### ISSUE TITLE
Auto vote when submission is made

### ISSUE THREAD

**Issue by robin-drexler** [jsunconf/contriboot-old#53]
When someone enters a submission she, is probably already interested in it.

So why not autovote?
We might have to adjust the "Thanks" text on the voting button, since it might not be clear, what we're thanking for when the app has automatically voted.

**Comment by robertkowalski** [comment]
i am not a big fan of autovoting:

i don't vote for my own suggestions and i would really get curious if i would get a vote automatically after creation.

i can't really see a real benefit while it would also make our code quite more complex

**Comment by robin-drexler** [comment]
I disagree, at least for interests. I think If you propose an interest you're most likely interested in it and that what's the button is for.

But yeah, it's just an idea and you're right it might seem a little bit odd for contributions.
